app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLocked/index.tsx

```diff
@@ -56,6 +56,19 @@ const probesetToMapping = (
   return newMap;
 };
 
+const otherMetricsToMapping = (
+  otherMetrics: { [metric: string]: string },
+  metricsMetdata: { [metric: string]: MetadataPoint },
+) => {
```

Comment:
Nit: can we name this something like otherMetricsToFriendlyName to make it more clear what it's mapping? I realize you're following the pattern already set with probesetToMapping but I think that could be more clear too (outside scope here). 🤔

Also, I think `metricsMetdata` should be `metricsMetaData`.